en/flight_controller/cuav_v5_nano.md

```diff
@@ -140,61 +140,44 @@ The *V5 nano* has no over current protection.
 Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos.
 The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
 
+## Compatibility 
 
-## Known Issues
-
-The issues below refer to the *batch number* in which they first appear. 
-The batch number is the last two digits of the *serial number* that appears on a sticker on the side of the flight controller. 
-For example, the serial number V031907XXXX**01** indicates the flight controller was in batch 01.
+CUAV adopts some differentiated designs and is incompatible with some hardware, which will be described below.
 
-
-#### GPS not compatible with other devices (Critical) - Batch 01 {#issue_gps_compatible}
+#### GPS not compatible with other devices {#compatibility_gps}
 
 The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
+The GPS will not work with other flight controllers.UAVCAN GPS is ideal for avoiding this compatibility.
 
-The GPS will not work with other flight controllers, and is the only GPS unit that can be used with the *CUAV V5+* and *CUAV V5 nano*.
-<!-- Nano/80/V5_NANO_RC01_2019_05_29 -->
-
-- *Found:* Batch 01
-- *Fixed:* -
-
-#### PM2 cannot power the flight controller {#issue_pm2}
-
-`PM2` can only measure battery voltage and current, but **not** power the flight controller.
-
-> **Warning** We recommend you do not connect a power module to this connector when using PX4.
-
-- *Found:* Batch 01
-- *Fixed:* -
+#### Using JTAG for hardware debugging {#compatibility_jtag}
 
-#### Volt regulation varies greater than +/- 5%
```

Comment:
FYI From @cuhome This was deleted because it is a feature. See:

> delete the voltage adjustment change greater than +/- 5%, connect the USB motherboard voltage is 4.6V; the reason is that the USB voltage is too low (close to 4.6v); connect Battery, the main board voltage is 5.2V, because HV_PM outputs 5.3V; This is a special design of CUAV and is within the normal operating voltage range of the flight controller.